### PR TITLE
Document /packit verify-fas

### PR DESCRIPTION
**Pull request by lbarcziova**
Merge when this is in production.

### CODE REVIEW

**Review by TomasTomecek — APPROVED** [review]
Could you please mention this requirement over here? https://packit.dev/docs/packit-service/#requirements-for-running-packit-service-jobs

Probably link from there to this question in FAQ.

I was thinking if a screenshot of the FAS admin interface help here. I'm going to configure this for my FAS right now.

File: content/docs/faq.md [hunk 1]
```diff
@@ -11,13 +11,20 @@ aliases: [/faq]
...
-[allowlist tracker](https://github.com/packit/notifications/issues/)
-to resolve this.
 Just be aware that we are now onboarding Fedora contributors who have
 a valid [Fedora Account System](https://fedoraproject.org/wiki/Account_System) account.
-So provide us enough information (your FAS account) in the issue.
-Until that, you will get a neutral status with `Namespace is not allowed!` message on your commits.
+
+We require our new users to have `GitHub Username` field set in the FAS account so that we can automatically
+check for the match between the `GitHub Username` field in the  
+FAS account and the Github account that triggers the verification.
```
Comment (csomh) [inline content/docs/faq.md:19]: Use `GitHub` (CamelCase) for consistency.

File: content/docs/faq.md [hunk 2]
```diff
@@ -11,13 +11,20 @@ aliases: [/faq]
...
-to resolve this.
 Just be aware that we are now onboarding Fedora contributors who have
 a valid [Fedora Account System](https://fedoraproject.org/wiki/Account_System) account.
-So provide us enough information (your FAS account) in the issue.
-Until that, you will get a neutral status with `Namespace is not allowed!` message on your commits.
+
+We require our new users to have `GitHub Username` field set in the FAS account so that we can automatically
+check for the match between the `GitHub Username` field in the  
+FAS account and the Github account that triggers the verification. 
+For each installation, we create a new issue in our
```
Comment (csomh) [inline content/docs/faq.md:20]: Does this happen in every case, or only when the GitHub and FAS usernames are different?

File: content/docs/faq.md [hunk 3]
```diff
@@ -11,13 +11,20 @@ aliases: [/faq]
...
+[allowlist tracker](https://github.com/packit/notifications/issues/)
+to resolve this, so you can just provide the FAS account in a comment
+as `/packit verify-fas my-fas-username` in the issue we create 
+for your installation. This will trigger the automatic check.
+Until that, you will get a neutral status with `Namespace is not allowed!`
```
Comment (csomh) [inline content/docs/faq.md:25]: Could this message be something like: `github.com/<namespace> needs to be approved before building.` And maybe link to this documentation.

File: content/docs/faq.md (hunk 2 above)
Comment (lbarcziova) [reply content/docs/faq.md]: Good point, I changed it a little bit, PTAL

**Comment by lbarcziova** [comment]
> Could you please mention this requirement over here? https://packit.dev/docs/packit-service/#requirements-for-running-packit-service-jobs
Probably link from there to this question in FAQ.

I moved it to the page you linked so that we don't create cycles in links :D PTAL

File: content/docs/faq.md (hunk 3 above)
Comment (lbarcziova) [reply content/docs/faq.md]: This was already adjusted in the past, but yes, this sounds more straightforward. I can open an issue in packit-service for this.

Comment (csomh) [reply content/docs/faq.md]: Thanks!

File: content/docs/faq.md (hunk 2 above)
Comment (csomh) [reply content/docs/faq.md]: Looks better, thanks!

**Review by csomh — APPROVED** [review]

**Pull request merged**